frontend/src/components/widgets/DataFrame/DataFrame.test.tsx

```diff
@@ -92,4 +96,24 @@ describe("DataFrame widget", () => {
     expect(dataFrameContainer.size.width).toBe(400)
     expect(dataFrameContainer.size.height).toBe(400)
   })
+
+  it("Touch detection correctly deactivates some features", () => {
```

Comment:
For this test, would it be possible to test it with jest? Might be a good idea to just try it out since we are technically trying to migrate from enzyme to jest. However, if it's too hard, then we can revisit this test maybe?

Reply:
Good point 👍 It's probably quite simple, but I think I will do that as a separate task since there are lots of very similar tests in this file and a few other dataframe-related tests. I will probably do a rewrite of everything cypress -> react-testing-library related to the dataframe component in one PR later.

Reply:
Sounds good. We can do that in a separate PR then for now. LGTM then

Reply:
I tracked this task on notion